### ISSUE TITLE
Is there any chance to change working directory?

### ISSUE THREAD

**Issue by DF-18** [mattiasnordin/StataEditor#41]
Hi,  Mattias
Thank you for sharing this plugin.
Here is an issue i m trying to fix.  When I run the code in sublime like this, the file will be output in the Sublime installation path. How can I set the output path to  "C:\abc", the same path as the dataset used?
`use "C:\abc\a.dta" 

reg y x

outreg2 using test.doc,replace`

Or, I have to add "cd" command to every do-file that use different dataset in different path?

**Comment by mattiasnordin** [comment]
Yes, you can use the setting "default_path" in the settings file.
